OSDOCS-16939: DITA abstracts — OLM understanding topics (#115281)
openshift/openshift-docs: modules/olm-operatorgroups-membership.adoc

[id="olm-operatorgroups-membership_{context}"]
= Operator group membership

[role="_abstract"]

> An Operator becomes a member of an Operator group when its CSV is in the same namespace and its install modes support the group's target namespaces. CSV install mode types define whether an Operator can watch its own namespace, one namespace, multiple namespaces, or all namespaces.

Reviewer: ```suggestion
An Operator becomes a member of an Operator group when its cluster service version (CSV) is in the same namespace and its install modes support the group's target namespaces.
```
